framework/tlvf/src/src/ClassList.cpp

```diff
@@ -34,6 +34,8 @@ void ClassList::swap()
     // call all tlv finalize functions
     for (auto const &c : m_class_vector) {
         c->class_swap();
+        if (c->getInnerClassList())
+            c->getInnerClassList()->swap();
```

Comment:
Actually - now that parse() does not automatically parse the inner class list, it isn't swapped, so this commit can be removed altogether - the backhaul manager parse OR the notorious print_cmdu_types method will simply not parse the inner list. I'm debating weather we should keep this here for the future in case we do the parsing then decide to swap back for forwarding. What do you think?